app/assets/stylesheets/components/common/_newsletter_signup.scss

```diff
@@ -22,28 +27,18 @@
   @include body(16, 16);
   border: 1px solid $color-newsletter-input-border;
   height: auto;
-  padding: $baseline-unit*2;
-  margin-right: $baseline-unit;
-  margin-bottom: $baseline-unit*3;
-  max-width: 238px;
-
-  @include respond-to($mq-l, $mq-xl) {
-    max-width: 165px;
-  }
-
-  .theme-cy & {
-    max-width: 208px;
-
-    @include respond-to($mq-l, $mq-xl) {
-      max-width: 135px;
-    }
-  }
+  padding: $baseline-unit*1.5 $baseline-unit*2;
+  margin: 0 auto $baseline-unit*2;
+  max-width: 268px; // 16.8rem;
```

Comment:
You don't have to use this here, but we have a function called `px` that will generate rems or pixel fallback. So you'd use it like this:

```scss
max-width: px(268);
```

But yeah, up to you whether you think it's worth it here